addons/mrp/static/src/widgets/mrp_product_configurator.js

```diff
@@ -0,0 +1,38 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { many2OneField } from "@web/views/fields/many2one/many2one_field";
+import { ProductField } from "@product/js/product_configurator/product_configurator_field";
+
+
+export class MrpProductField extends ProductField {
+    // This class existence is only justified by the fact that some fields do not have the same name
+    // in mrp than in the other modules using the configurator.
```

Comment:
In this case, instead of an extend the class, can we have some parameters instead to define those fields' name ?
You should be able to define some props from the view thank to `extractProps`